swig/python/README.rst

```diff
@@ -122,6 +122,28 @@ Verify that the necessary dependencies have been installed and then run the foll
     pip install --no-cache --force-reinstall gdal[numpy]=="$(gdal-config --version).*"
 
 
+uv
+~~~
+
+Uv build local wheel for GDAL < 3.9 specific configuration must be added to the `pyproject.toml` to add Numpy to the build environnment prior to build gdal wheel.
```

Comment:
I've never heard about "uv" before, so I'm not sure to understand that section. Is it in an addition to GDAL's pyproject.toml ? Would there be any harm to add it to the official version, even if users don't use uv ?

Reply:
Hi,

first of all Happy New Year to all!

I did look at the GDAL's pyproject.toml. My PR try to give information on how to add gdal as dependencie to project/script managed with uv. I Personally lost some time to do so and I happy to share it hear.

In my personal case I try to create a web service based on gdal and fastAPI, uv make a geat work to manage versoins. however my debian 12 have limitation on GDAL's version (3.6.2) and do not have late development.

I'll take a look to the pyproject.toml if this treaks would help

regards